docs/en/reference/configuration.rst

```diff
@@ -155,6 +156,15 @@ pdo_sqlite
    in-memory (non-persistent). Mutually exclusive with ``path``.
    ``path`` takes precedence.
 
+sqlite3
+^^^^^^^
+
+-  ``path`` (string): The filesystem path to the database file.
+   Mutually exclusive with ``memory``. ``path`` takes precedence.
+-  ``memory`` (boolean): True if the SQLite database should be
+   in-memory (non-persistent). Mutually exclusive with ``path``.
+   ``path`` takes precedence.
```

Comment:
The "`path` takes precedence" clause is no longer relevant.

Reply:
Right.

Reply:
🙈